app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryDescription.js

```diff
@@ -35,21 +41,29 @@ class ExpandedStoryDescription extends React.Component {
     );
   };
 
-  descriptionTextArea(description) {
-    const { onEdit, disabled } = this.props;
+  descriptionTextArea(description, suggestedUsers) {
+    const { disabled, onEdit } = this.props;
```

Comment:
Since we only need the suggested users here, what do you think about doing that `map` here instead of directly inside render? You can destructure `users` from props and well and create the `suggestedUsers` here.